swarm-event-schema.json

```diff
@@ -4,7 +4,7 @@
     "title": "Swarm Event Schema",
     "definitions":
     {
-        "uuid":
+        "id":
```

Comment:
What is id?

Reply:
A (used in other places) type definition.